src/screens/Landing.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
```

Comment:
instead of screens it might be `pages` and for example `dropdown` is a component so you can create one folder for components called `components` and move all your components here